vendor/bat-native-ledger/src/bat/ledger/internal/bat_publishers.cc

```diff
@@ -634,13 +634,13 @@ BatPublishers::getAllBalanceReports() {
   for (auto const& report : state_->monthly_balances_) {
     ledger::BalanceReportInfo newReport;
     const braveledger_bat_helper::REPORT_BALANCE_ST oldReport = report.second;
-    newReport.opening_balance_ = oldReport.opening_balance_;
-    newReport.closing_balance_ = oldReport.closing_balance_;
-    newReport.grants_ = oldReport.grants_;
-    newReport.earning_from_ads_ = oldReport.earning_from_ads_;
-    newReport.auto_contribute_ = oldReport.auto_contribute_;
-    newReport.recurring_donation_ = oldReport.recurring_donation_;
-    newReport.one_time_donation_ = oldReport.one_time_donation_;
+    newReport.opening_balance = oldReport.opening_balance_;
```

Comment:
rename `BatPublishers::getAllBalanceReports()` -> `BatPublishers::GetAllBalanceReports()`

change return `std::map<std::string, ledger::BalanceReportInfo>` -> `std::map<std::string, ledger::BalanceReportInfoPtr>` (this way you don't need to convert it to ptr in `bat_ledger_impl`